evergreen/evergreen.yml

```diff
@@ -492,6 +496,13 @@ buildvariants:
   tasks:
      - name: "test"
 
+- matrix_name: "tests-zstandard-compression"
+  matrix_spec: { compressor : "zstandard", auth: "noauth", ssl: "nossl", version: ["3.6", "4.0", "4.2", "latest"], topology: "standalone", os: "*" }
```

Comment:
I believe we need "4.4" in here as well.

Reply:
Yep, I've planned to add it in merging with master, but i'm ok to do it now too

Reply:
done